<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="AjayNamala" failures="1" tests="6" name="smoke.LoginTest" time="46.884" errors="0" timestamp="2025-08-15T11:56:46 IST" skipped="0">
  <testcase classname="smoke.LoginTest" name="login" time="6.341">
    <failure message="You landed into incorrect page expected [login failed! please ensure the username and password are val] but found [login failed! please ensure the username and password are valid.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: You landed into incorrect page expected [login failed! please ensure the username and password are val] but found [login failed! please ensure the username and password are valid.]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at pages.LoginPage.userLogin(LoginPage.java:68)
at smoke.LoginTest.login(LoginTest.java:20)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
    </failure>
  </testcase> <!-- login -->
  <system-out/>
  <testcase classname="smoke.LoginTest" name="login" time="6.595"/>
  <system-out/>
  <testcase classname="smoke.LoginTest" name="login" time="13.641"/>
  <system-out/>
  <testcase classname="smoke.LoginTest" name="login" time="7.624"/>
  <system-out/>
  <testcase classname="smoke.LoginTest" name="login" time="6.269"/>
  <system-out/>
  <testcase classname="smoke.LoginTest" name="login" time="6.414"/>
  <system-out/>
</testsuite> <!-- smoke.LoginTest -->
